assets/training/finetune_acft_hf_nlp/src/model_selector/model_selector.py

```diff
@@ -421,7 +441,7 @@ def model_selector(args: Namespace):
                     mlflow_data = yaml.safe_load(fp)
                 if mlflow_data and "flavors" in mlflow_data:
                     for key in mlflow_data["flavors"]:
-                        if key in ["hftransformers", "hftransformersv2"]:
```

Comment:
Is this same logic applicable to OSS flavor as well? Are we going to save the same parameters in OSS MLModel file as we are doing for hftransformers MLModel file?

Reply:
yes